tests/posit/connect/test_users.py

```diff
@@ -109,14 +108,14 @@ def test_users_find(self):
             "https://connect.example/__api__/v1/users",
             match=[
                 responses.matchers.query_param_matcher(
-                    {"page_size": 2, "page_number": 2}
+                    {"page_size": 500, "page_number": 2}
                 )
             ],
             json=load_mock("v1/users?page_number=2&page_size=2.json"),
```

Comment:
Do you want to update the file names to reflect the changed query params?

Reply:
Thanks! I cleaned this up.